tests/test_ref.py

```diff
@@ -650,3 +656,12 @@ def test_change_spec(self, spec):
         with pytest.raises(KeyError):
             after[270]
         assert list(after.find_all(270)) == [[268, 0, 270], [268, 1, 270]]
+
+    def test_serialisation_header(self, spec):
+        if 'FIX5' in spec.version:
```

Comment:
If condition could be remove because each version of fix spec should support tag order of header.